bftengine/src/bftengine/CheckpointInfo.hpp

```diff
@@ -42,6 +42,8 @@ class CheckpointInfo {
 
   bool isCheckpointCertificateComplete() const;
 
+  bool isCheckpointSuperStable() const;
```

Comment:
Can you please add a comment indicating what it means for a checkpoint to be super stable?

Reply:
done